components/keyrings/common/config/config_reader.cc

```diff
@@ -25,17 +25,38 @@
 
 #include "config_reader.h"
 
+#include <rapidjson/error/en.h>
 #include <rapidjson/istreamwrapper.h> /* IStreamWrapper */
+#include <stdlib.h>
+#include <unistd.h>
+#include <iostream>
 
 namespace keyring_common {
 namespace config {
 
 Config_reader::Config_reader(const std::string config_file_path)
     : config_file_path_(config_file_path), data_(), valid_(false) {
+  sleep(1);
   std::ifstream file_stream(config_file_path_);
-  if (!file_stream.is_open()) return;
+  if (!file_stream.is_open()) {
+    fprintf(stderr, "Keyring configuration doesn't exists: %s\n",
```

Comment:
The issue was that we can't use LogComponentErr in the common code, because we don't have a fixed component there. I've moved this to the header to workaround that.